Library/UIView+AutolayoutTests.swift

```diff
@@ -6,11 +6,49 @@ final class UIViewAutoLayoutExtensionTests: TestCase {
     let view1 = UIView(frame: .zero)
     let view2 = UIView(frame: .zero)
 
-    _ = (view2, view1)
+    let targetView = UIView(frame: .zero)
 
-    _ = ksr_insertSubviewInParent(at: 0)(view2, view1)
+    targetView.addSubview(view1)
```

Comment:
Could this use `ksr_addSubviewToParent`?

Reply:
It could, but because we're testing one very specific `ksr` helper I don't want to couple another `ksr` function to these tests.